feather/base/src/anvil/player.rs

```diff
@@ -71,7 +71,7 @@ pub struct InventorySlot {
 
 impl InventorySlot {
     /// Converts an `ItemStack` and network protocol index into an `InventorySlot`.
```

Comment:
Technically you could link them with `[`ItemStack`]` (and the other one) but its not a requirement.